src/components/Drawer/__test__/drawer.a11y.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { axe } from 'jest-axe';
+import Drawer from '..';
+import Button from '../../Button';
+
+describe('<Drawer/>', () => {
+    it('should be accessible', () => {
+        const wrapper = mount(
+            <Drawer isOpen title="Drawer a11y" footer={<Button label="button in drawer footer" />}>
+                <p>A rainbow is a meteorological phenomenon ...</p>
+            </Drawer>,
+        );
+        setTimeout(async () => {
+            expect.assertions(1);
+            const html = wrapper.html();
+            const results = await axe(html);
+            expect(results).toHaveNoViolations();
+        }, 500);
```

Comment:
```suggestion
    Promise.resolve().then(() => {
        expect.assertions(1);
        const html = wrapper.html();
        const results = await axe(html);
        expect(results).toHaveNoViolations();                
    });
```

Reply:
same for all the setTimeout, it should work with the promise resolving in the next tick

Reply:
if setTimeout is totally necessary because of the animation of the drawer then you should use the `done` function inside of the setTimeout

```js
it('should', (done) => {
    ...
    setTimeout(() => {
        ...
        done();
    }, 500);
});
```

Reply:
yes i think i will use the done approach as long as the setTimeout is mandatory due to the animation...

Reply:
by using the promise approach on accessibility test.. it failed... and also fail with the done approach... and it fails because "wrapper.html()" returns null... not sure why...  the only way it runs is the way i implement it... not sure what i am doing wrong..